<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>1021</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Logo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(255,255,255);
border: 0;
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myData.qrc">
        <normaloff>:/icons/HTI.jpg</normaloff>:/icons/HTI.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>401</width>
        <height>394</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_Title">
         <property name="styleSheet">
          <string notr="true">color: #0A66C2;
font-size: 28px;
font: &quot;Segoe UI&quot;;
font-weight: 700;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
padding-left: 20px;
padding-right: 20px;
align: center;
text-align: center;
vertical-align: middle;
</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Black Jack&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_BJ_Logo">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(255,255,255);
border: 0;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="myData.qrc">
           <normaloff>:/icons/black_jack.png</normaloff>:/icons/black_jack.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_play">
         <property name="styleSheet">
          <string notr="true">background-color: #0A66C2;
border: 0;
border-radius: 15px;
color: #ffffff;
font-family: -apple-system, system-ui, system-ui, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, &quot;Fira Sans&quot;, Ubuntu, Oxygen, &quot;Oxygen Sans&quot;, Cantarell, &quot;Droid Sans&quot;, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Lucida Grande&quot;, Helvetica, Arial, sans-serif;
font-size: 16px;
font-weight: 600;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
padding-left: 20px;
padding-right: 20px;
text-align: center;
vertical-align: middle;
</string>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_help">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>500</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(255,255,255);
border: 0;
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myData.qrc">
        <normaloff>:/icons/exclamation-mark.png</normaloff>:/icons/exclamation-mark.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Room</string>
     </attribute>
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>511</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0A66C2;
font-size: 28px;
font: &quot;Segoe UI&quot;;
font-weight: 700;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
padding-left: 20px;
padding-right: 20px;
align: center;
text-align: center;
vertical-align: middle;
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Draw a Card&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>981</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,0,0,0,10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_p1" stretch="1,1,20,2,2">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: #0A66C2;
font: 700 24pt &quot;Segoe UI&quot;;
line-height: 25px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
padding-left: 20px;
padding-right: 20px;
text-align: center;;
vertical-align: middle;
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Player 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_p1_score">
           <property name="styleSheet">
            <string notr="true">color: #0A66C2;
font-family: -apple-system, system-ui, system-ui, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, &quot;Fira Sans&quot;, Ubuntu, Oxygen, &quot;Oxygen Sans&quot;, Cantarell, &quot;Droid Sans&quot;, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Lucida Grande&quot;, Helvetica, Arial, sans-serif;
font-size: 16px;
font-weight: 600;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
text-align: left;
vertical-align: middle;
</string>
           </property>
           <property name="text">
            <string>Score:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_p1">
           <property name="styleSheet">
            <string notr="true">image: url(:/icons/card.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_p1_draw">
           <property name="styleSheet">
            <string notr="true">
QPushButton {
   background-color: #0A66C2;
border: 0;
border-radius: 15px;
color: #ffffff;
font: 625 19pt &quot;Segoe UI&quot;;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
padding-left: 20px;
padding-right: 20px;
text-align: center;
vertical-align: middle;
}

QPushButton:disabled{
background-color: rgb(161, 161, 161);
}
</string>
           </property>
           <property name="text">
            <string>Draw</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_p1_pass">
           <property name="styleSheet">
            <string notr="true">
QPushButton {
   background-color: #0A66C2;
border: 0;
border-radius: 15px;
color: #ffffff;
font: 625 19pt &quot;Segoe UI&quot;;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
padding-left: 20px;
padding-right: 20px;
text-align: center;
vertical-align: middle;
}

QPushButton:disabled{
background-color: rgb(161, 161, 161);
}
</string>
           </property>
           <property name="text">
            <string>Pass</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_playAgain">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton {
   background-color: rgb(155, 89, 255);
border: 0;
border-radius: 15px;
color: #ffffff;
font: 625 19pt &quot;Segoe UI&quot;;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
padding-left: 20px;
padding-right: 20px;
text-align: center;
vertical-align: middle;
}

QPushButton:disabled{
background-color: rgb(161, 161, 161);
}
</string>
         </property>
         <property name="text">
          <string>Play Again</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_p2" stretch="1,1,20,2,2">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">color: #B22222;
font: 700 24pt &quot;Segoe UI&quot;;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
padding-left: 20px;
padding-right: 20px;
text-align: center;

vertical-align: middle;
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Player 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_p2_score">
           <property name="styleSheet">
            <string notr="true">color: #B22222;
font-family: -apple-system, system-ui, system-ui, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, &quot;Fira Sans&quot;, Ubuntu, Oxygen, &quot;Oxygen Sans&quot;, Cantarell, &quot;Droid Sans&quot;, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Lucida Grande&quot;, Helvetica, Arial, sans-serif;
font-size: 16px;
font-weight: 600;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
text-align: left;
vertical-align: middle;
</string>
           </property>
           <property name="text">
            <string>Score:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_p2">
           <property name="styleSheet">
            <string notr="true">image: url(:/icons/card.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_p2_draw">
           <property name="styleSheet">
            <string notr="true">
QPushButton {
   background-color: #B22222;
border: 0;
border-radius: 15px;
color: #ffffff;
font: 625 19pt &quot;Segoe UI&quot;;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
padding-left: 20px;
padding-right: 20px;
text-align: center;
vertical-align: middle;
}

QPushButton:disabled{
background-color: rgb(161, 161, 161);
}
</string>
           </property>
           <property name="text">
            <string>Draw</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_p2_pass">
           <property name="styleSheet">
            <string notr="true">
QPushButton {
   background-color: #B22222;
border: 0;
border-radius: 15px;
color: #ffffff;
font: 625 19pt &quot;Segoe UI&quot;;
line-height: 20px;
max-width: 480px;
min-height: 40px;
min-width: 0px;
padding: 0px;
padding-left: 20px;
padding-right: 20px;
text-align: center;
vertical-align: middle;
}

QPushButton:disabled{
background-color: rgb(161, 161, 161);
}
</string>
           </property>
           <property name="text">
            <string>Pass</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1319</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="myData.qrc"/>
 </resources>
 <connections/>
</ui>
